templates/citation-styles/ris.tpl

```diff
@@ -16,20 +16,73 @@
  *}
 {assign var="containerTitle" value="container-title"}
 {assign var="containerTitleShort" value="container-title-short"}
-TY  - JOUR
+{assign var="collectionTitle" value="collection-title"}
+{assign var="collectionEditor" value="collection-editor"}
+{assign var="publisherPlace" value="publisher-place"}
```

Comment:
It looks like these three `{assign ...}` variables are not used?

Reply:
`collectionTitle` is used in line 57 & 58, 
`collectionEditor` is used in line 27, 28, 37 & 38,
`publisherPlace` is used in line 63.